portal-web/test/functional/com/liferay/portalweb/tests/enduser/changelist/PublishChangeList.testcase

```diff
@@ -34,6 +34,73 @@ definition {
 		}
 	}
 
+	@priority = "5"
+	test ConflictingChangesAutomaticallyResolved {
+		property test.name.skip.portal.instance = "PublishChangeList#ConflictingChangesAutomaticallyResolved";
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Web Content");
+
+		WebContentNavigator.gotoAddCP();
+
+		WebContent.addCP(
+			webContentContent = "WC WebContent Content",
+			webContentTitle = "WC WebContent Title");
+
+		PortletEntry.publish();
+
+		ChangeList.createNewPublication(publicationName = "Publication Name");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Web Content");
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "WC WebContent Title");
+
+		WebContent.editCP(
+			webContentContentEdit = "WC WebContent Content Edit",
+			webContentTitle = "WC WebContent Title",
+			webContentTitleEdit = "WC WebContent Title Edit");
+
+		ChangeList.createNewPublication(publicationName = "Publication 2 Name");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Web Content");
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "WC WebContent Title");
+
+		WebContent.editCP(
+			webContentContentEdit = "WC WebContent Content 2 Edit",
+			webContentTitle = "WC WebContent Title",
+			webContentTitleEdit = "WC WebContent Title 2 Edit");
+
+		ChangeList.prepareToPublish(publicationName = "Publication 2 Name");
+
+		ChangeList.publishPublication(publicationName = "Publication 2 Name");
+
+		ChangeList.selectPublication(publicationName = "Publication Name");
+
+		ChangeList.prepareToPublish(
+			gotoManagePublications = "true",
+			publicationName = "Publication Name");
+
+		AssertElementNotPresent(locator1 = "ChangeList#NEEDS_MANUAL_RESOLUTION_TABLE");
+
+		AssertElementNotPresent(
+			key_warningMessage = "The conflict cannot be automatically resolved.",
+			locator1 = "Message#WARNING_SPECIFIC");
```

Comment:
I would be wary about only using negative assertions. Should we assert the `Modification Conflict: The conflict was automatically resolved` alert as well?

Also we should add a test that clicks the `View` button on the prepare to publish screen. Do you think it would fit here?

Reply:
Hi @xbrianlee, the reason why I didn't assert the `Modification Conflict: The conflict was automatically resolved` is that currently automatically resolved is blocked by LPS-115599, and it doesn't determine. Actually I'd add a macro to verify which change is automatically resolved, which change needs manual resolution later. And For `View` I will add it to the tests together with the macro I mentioned above. Do you think it's available?

Reply:
And now, I just add assertation `Modification Conflict: The conflict was automatically resolved` alert as well.

Reply:
Sounds good, that makes more sense 👍